server/profile.py

```diff
@@ -0,0 +1,29 @@
+from app import db
+
+class Profile(db.Model):
+    __tablename__ = 'profile'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(120), nullable=False)
+    image = db.Column(db.Text)
```

Comment:
we can have a more descriptive name like `profile_image` or `profile_picture_url`, etc.

Reply:
i'll update the name fields